examples/test_interpose_multiple.c

```diff
@@ -158,18 +159,17 @@ int main (int argc, char* argv[])
   // prime system once before timing
   int t;
   for(t=0; t < 1; t++) {
-    int rc;
   for (i=0; i < num_files; i++) {
     char* file = files[i];
     int fd_me = open(file, O_WRONLY | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR);
     if (fd_me > 0) {
       // write the checkpoint
-      rc = write_checkpoint(fd_me, timestep, bufs[i], filesizes[i]);
+      write_checkpoint(fd_me, timestep, bufs[i], filesizes[i]);
 
-      rc = fsync(fd_me);
+      fsync(fd_me);
 
       // make sure the close is without error
```

Comment:
this comment is a lie (even more so now than before) since no such making sure (ie return code checking) happens.

Reply:
Fixed in latest push